diesel/src/expression/functions/date_and_time.rs

```diff
@@ -23,13 +23,9 @@ impl<DB: Backend> QueryFragment<DB> for now {
         Ok(())
     }
 
-    fn collect_binds(&self, _out: &mut DB::BindCollector) -> QueryResult<()> {
+    fn walk_ast(&self, _pass: &mut AstPass<DB>) -> QueryResult<()> {
```

Comment:
Because I wrote the first third of this, stashed it, wrote three other prototypes which all turned out to be dead ends, rewrote a sixth of this, remembered that I had a stashed partial impl of this version, and had merge conflicts because I switched styles in the second one. I'll fix it lol